srtcore/group.cpp

```diff
@@ -1223,12 +1223,6 @@ int CUDTGroup::sendBroadcast(const char* buf, int len, SRT_MSGCTRL& w_mc)
     // and therefore take over the leading role in setting the ISN. If the
     // second one fails, too, then the only remaining idle link will simply
     // go with its own original sequence.
-    //
-    // On the opposite side the reader should know that the link is inactive
-    // so the first received payload activates it. Activation of an idle link
-    // means that the very first packet arriving is TAKEN AS A GOOD DEAL, that is,
-    // no LOSSREPORT is sent even if the sequence looks like a "jumped over".
```

Comment:
It's not true after https://github.com/Haivision/srt/pull/2257

Reply:
Then please replace it with a comment that explains what really happens in this case.

The mechanism of joining a new group member during the transmission relies on that the ISN must be taken from the exact packet that is being sent at the moment, but then some time passes between the handshake and readiness for transmission. Packets in this sequence range will never be sent and should never be retransmission-requested. Therefore, if we don't forcefully hide this initial portion, there must be some other mechanism that ensures they will not be forcefully uselessly retransmitted. Or, maybe not, in which case this forced and useless retransmission will happen to ensure that when some of packets were lost on the existing and soon broken link, they will still be recovered.